### ISSUE TITLE
About the object detection part of the paper

### ISSUE THREAD

**Issue by 1seyana1** [runjia0124/CoCoNet#11]
May I ask about the m3fd dataset, how exactly did you divide it and did you retrain or fine-tune the fusion part of this dataset? I can't achieve the results in the paper using yolov5s.m3fd in detection are you using the weights provided in the code directly? Do I need to retrain it?
Thank you for your work!

**Comment by runjia0124** [comment]
> May I ask about the m3fd dataset, how exactly did you divide it and did you retrain or fine-tune the fusion part of this dataset? I can't achieve the results in the paper using yolov5s.m3fd in detection are you using the weights provided in the code directly? Do I need to retrain it? Thank you for your work!

Hello, 
sorry for getting back late, (travel:) We did not finetune the fusion part, it is fixed. We used the fusion weights provided here, actually training a new fusion model should not have a big difference. The detection part is retrained on the fused images. You need to retrain the detection part since the original yolo is trained on natural images. Was the fused image intensity normalized before feeding into the detection model? Feel free to let me know if you have more questions.
